Assignment2/gemawari-GemaGonzalez.ttl

```diff
@@ -0,0 +1,16 @@
+@base <http://www.oeg-upm.net/ontologies/assignment2/>
+@prefix vocab: <http://www.ontologies.org/ontologies/assignment2#>
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Class01 vocab:includes :Sensor029;
+		 vocab:includes :Computer101.
+
+:Sensor029 vocab:hasMeasurement :Measurement8401.
+
+:Measurement8401 vocab:hasTemperature "29"^^xsd:integer;
+				 vocab:atTime :2010-06-12T12:00:12^^xsd:dateType .

```

Comment:
The dateTime should be between " " and the initial : does not make sense

Reply:
.ttl of Assignment 2 was corrected (dateTime)